pkg/cluster/cluster.go

```diff
@@ -471,7 +484,8 @@ func GetClusterAddOns(connection *sdk.Connection, clusterID string) ([]*AddOnIte
 
 			// Only display add-ons for which the org has quota
 			resourceQuotas.Each(func(resourceQuota *amsv1.ResourceQuota) bool {
-				if addOn.ResourceName() == resourceQuota.ResourceName() {
+				addOnPrefix := "add-on|addon-"
+				if addOnPrefix+addOn.ID() == skuRulesMap[resourceQuota.SKU()] {
```

Comment:
Replace `resourceQuotas` with QuotaCost, per the Slack conversation.  You can build the list of addons via QuotaCost. You don't need ResourceQuota or SKUs.